sbin/xivo-check-master-status

```diff
@@ -2,15 +2,28 @@
 ip_addr=$1
 ping_count="3"
 ping_interval="10"
+confd_port=${CONFD_PORT:-9486}
+confd_version=${CONFD_API_VERSION:-1.1}
 
 if [ -z "$ip_addr" ]; then
     echo "usage: $(basename $0) ip_addr"
     exit 1
 fi
 
+check_ha_mode(){
+    local token=$(wazo-auth-cli token create --expiration 20)
+    curl -k -H "X-Auth-Token: $token" http://localhost:$confd_port/$confd_version/ha | jq -r .node_type
+}
```

Comment:
After discussion with François, this method is too heavyweight for its purpose: this creates a new token every minute and requires wazo-auth and wazo-confd to be constantly running (they should be, but we want to minimize impact if they don't). We suggest using a file `/var/lib/wazo/is-secondary` that would be present only when the instance is configured as a secondary stack ("slave"). The responsibility to create/erase this file would be for wazo-sysconfd. wazo-sysconfd already writes some files around, so it shouldn't be a big change.

Any objection?

Reply:
Done: https://github.com/wazo-platform/wazo-sysconfd/pull/72